contracts/schemes/ContinuousLockingToken4Reputation.sol

```diff
@@ -281,6 +290,7 @@ contract ContinuousLocking4Reputation is Agreement {
         return batches[_batchIndex].scores[_lockingId];
     }
 
+    // consider moving the Realmath functions to a separate file (or use a library???)
```

Comment:
yes. 
we have that on infra.
though still need to expose some funcs which are private.